libraries/psibase/common/include/psibase/blob.hpp

```diff
@@ -4,16 +4,29 @@
 #include <algorithm>
 #include <type_traits>
 
+template <typename T>
+concept IsByteType =
+    std::is_same_v<std::decay_t<T>, char> || std::is_same_v<std::decay_t<T>, unsigned char>;
+
+template <typename T>
+concept IsContainerLike = requires(T t) {
```

Comment:
This name is too generic for what it actually checks. Actually, if we're going to use concepts, it should be `std::ranges::contiguous_range<T> && std::ranges::sized_range<T>` (and use `std::ranges::data`/`std::ranges::size`).

Reply:
This should now be addressed, what do you think?